program/convert-weeks-to-days/ConvertWeeksToDays.js

```diff
@@ -0,0 +1,4 @@
+let weeks = 5;
+let day = weeks * 7;
+
```

Comment:
can you change the variable name to 'days' , so that it will be appropriate

Reply:
ok